src/styles/index.scss

```diff
@@ -1,3 +1,140 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
 }
+
+ul {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+ul li {
+  padding: 0;
+  margin: 0;
+}
+
+.card {
+  width: 200px;
```

Comment:
Try not use static width/height in places where their is not critical need to use.